apps/dashboard/config/configuration_singleton.rb

```diff
@@ -355,7 +351,7 @@ def ood_bc_card_time
   end
 
   def config
-    @config ||= read_config
+    @config = read_config
```

Comment:
I think this `||=` is still needed. We do need a way to refresh the configs, but I don't want to be reading the file every time we try to fetch stuff.

Reply:
Yes, I updated for local testing and missed reverting the changes before making the commit.